.circleci/config.yml

```diff
@@ -52,15 +53,15 @@ commands:
 jobs:
   melodic:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-2004:202111-02
```

Comment:
Is this portion of CI supposed to run on 1804?

Reply:
The `image` specified here is just used to install kvaser libs and run docker containers, so it's not too important what version it is.